meta.json

```diff
@@ -0,0 +1,27 @@
+{
+  "maintainers": [
+    {
+      "npm": "georgegillams",
+      "github": "georgegillams"
+    },
+    {
+      "npm": "k0nserv",
+      "github": "k0nserv"
+    },
+    {
+      "npm": "shaundon",
+      "github": "shaundon"
+    },
+    {
+      "npm": "tiagohngl",
+      "github": "tiagoengel"
+    },
+    {
+      "github": "willhg"
```

Comment:
😭

Reply:
Just realised @bvitaliyg is not in the file 😬. Will add it.